src/store/actions/onboarding.actions.ts

```diff
@@ -44,6 +47,32 @@ export const finalizeOnboarding = (): ThunkAction<Promise<void>, RootState, unkn
             identifier: contactIdentifier
           })
         )
+        dispatch(
+          createVerifiableCredential({
+            credential: {
+              "@context": [
+                "https://www.w3.org/2018/credentials/v1",
+                "https://www.w3.org/2018/credentials/examples/v1",
```

Comment:
Don't include the examples in here. Only the credentials/v1 and our context should be in here